languageFactories/language-factory-core/src/main/scala/cromwell/languages/LanguageFactory.scala

```diff
@@ -33,10 +33,11 @@ trait LanguageFactory {
     case _ => None.validNelCheck
   }
 
-  def getWomBundle(workflowSource: WorkflowSource,
-                   workflowOptionsJson: WorkflowOptionsJson,
-                   importResolvers: List[ImportResolver],
-                   languageFactories: List[LanguageFactory]): Checked[WomBundle]
+  def getWomBundleWithImports(workflowSource: WorkflowSource,
```

Comment:
I think you can actually still call this `getWomBundle` and not return the imports as a separate return value. Since the caller has passed in `importResolvers`, the caller is also able to ask the import resolvers for the set of imports that got resolved too.

Reply:
Actually yes. So somewhere when I was coding the first approach where I was retrieving the resolved imports after the wdl was validated, there was some problem because the `getWomBundle` was called repeatedly after resolving each input, and the previously resolved imports were lost. But here since the same `ResolvedImportsStore` [reference](https://github.com/broadinstitute/cromwell/blob/e7526f54022f53d2be58920fbe5b13ebfcc5077b/womtool/src/main/scala/womtool/input/WomGraphMaker.scala#L28) is passed to each new updated resolver, I can actually just retrieve the resolved imports set from that object rather than through the `importResolvers`. Thank you for pointing this out! :)